lib/server.js

```diff
@@ -65,7 +66,10 @@ class S3Server {
     constructor(worker) {
         this.worker = worker;
         this.cluster = true;
-        http.globalAgent.keepAlive = true;
+        http.globalAgent = new HttpAgent({
```

Comment:
Would it make sense to have this change on 7.4 branch as well? In which case, it needs its own ticket and PR (it can also be a backport later but it's cleaner with a forward-ported PR).

Reply:
it could, it was mainly for Azure client and the upgrade, not sure it affect anything else, but not cost much to that in 7.4